test/inductor/test_torchinductor.py

```diff
@@ -2452,6 +2458,21 @@ def fn(x, y):
 
         self.common(fn, [torch.randint(5, (1, 8)), 5400])
 
+    @torch._dynamo.config.patch(dynamic_shapes=True)
+    @torch._dynamo.config.patch(assume_static_by_default=False)
+    def test_scalar_output(self):
+        def fn(arg0_1, arg2_1):
+            arg1_1 = arg2_1.size(1)
+            view = torch.ops.aten.view.default(arg2_1, [-1, arg1_1])
+            embedding = torch.ops.aten.embedding.default(arg0_1, view)
+            full = torch.ops.aten.full.default([1, arg1_1], 1, dtype=torch.float32)
+            return (full, arg1_1, embedding)
+
+        arg0_1 = rand_strided((32128, 768), (768, 1), device="cpu", dtype=torch.float32)
+        arg1_1 = 22
```

Comment:
`arg1_1` is not used in this scope, delete it?

Reply:
Deleted the unused variable